flake.nix

```diff
@@ -51,31 +54,22 @@
       nixpkgsBySystem = forAllSystems (system: import nixpkgs {
         inherit system;
         overlays = [
-          self.overlay
+          self.overlays.default
           (final: prev: { update-nixified-deps = final.callPackage update-nixified-deps {}; })
         ];
       });
 
       # It's a shame this is necessary. Alas. Run this after updating the
       # castopod-host-src input: `nix run .#update-nixified-deps`
```

Comment:
Is this still true?

Reply:
Yes, this is still needed for php dependencies (dream2nix does not support php yet).